packages/e2e-tests/src/specs/editor/pendingStories.js

```diff
@@ -24,7 +24,8 @@ import {
   editStoryWithTitle,
 } from '@web-stories-wp/e2e-test-utils';
 
-describe('Pending Stories', () => {
+// eslint-disable-next-line jest/no-disabled-tests -- net::ERR_CONNECTION_RESET
+describe.skip('Pending Stories', () => {
```

Comment:
Needs a ticket to address this.

Also, does `retryTimes` help for this particular test?

Reply:
Removed - didn't seem to.